app/views/static_pages/index.html.haml

```diff
@@ -38,15 +38,15 @@
     member. Guests may be any gender or age.
   %p
     You can find out about upcoming events (and learn more about us) on #{link_to 'our blog', TUMBLR_URL},
-    following us on Twitter at #{external_link_to "@#{TWITTER_USERNAME}", TWITTER_URL}, or joining our #{link_to "general interest mailing list", MAILING_LIST_GENERAL}.
+    following us on Twitter at #{external_link_to "@#{TWITTER_USERNAME}", TWITTER_URL}, or joining our #{link_to "general interest mailing list", MAILING_LIST_GENERAL}. (Note: By clicking on the mailing list link, you will be redirected to our Google Groups page. Click "Join group" to get emails.)
```

Comment:
@mathildemouw Requesting a review from you to make sure this copy for the instructions is reasonable.

Reply:
Sweet, taking a look 👀